### DISCUSSION TITLE
Redis queue on manager node

### DISCUSSION THREAD

**Discussion by Sahale** [Security-Onion-Solutions/securityonion#5366] [Unsupported Versions] [answered]
Hi!
2.3.70 with hotfixes, manager node and some heavy nodes.
Redis queue on manager node slowly increases and doesn't process. As I see it's due to architecture (heavy nodes processes only their data). How can I clear redis queue?
Thanks.

**Comment by solarplex1** [comment]
I am having the same issue more or less. The problem is that the search index is grown beyond its capacity which put redis into read only mode. This has queued all of my logs and prevented them from reaching elastic search.

**Comment by dougburks** [comment, marked as answer]
@Sahale Did you follow the steps in the heavy node hotfix blog post?
https://blog.securityonion.net/2021/07/security-onion-2360-heavy-node-hotfix.html

**Reply by Sahale** [reply]
Yes. I can repeat this if needed.

**Reply by dougburks** [reply]
If you've verified that heavy nodes are processing their own logs locally and NOT sending logs to the manager, then the logs in redis on the manager must be originating from Wazuh or other local system logs on the manager itself. To avoid losing these logs, you might want to try adding a separate search node (NOT heavy node) to drain the manager's redis queue.

**Reply by Sahale** [reply]
Yes, this is local system log. Sorry that haven't noticed this earlier.
What if I want to drop this queue? Just ignore it and it'll be cleared automatically?

**Reply by dougburks** [reply]
You could manually clear the redis queue, but it will continue to fill as the manager generates new logs. To avoid losing these logs, you might want to try adding a separate search node (NOT heavy node) to drain the manager's redis queue. We've updated the documentation to reflect this:
https://docs.securityonion.net/en/2.3/architecture.html#distributed
